### PR TITLE
feat: fetch twts by username

### PR DESCRIPTION
**Pull request by dooven**
##### Summary
- Added /profile/:nick/twts

##### Component Name
- area/backend

### CODE REVIEW

File: internal/api.go [hunk 1]
```diff
@@ -68,13 +68,19 @@ func (a *API) initRoutes() {
...
+
 	router.POST("/follow", a.isAuthorized(a.FollowEndpoint()))
 	router.POST("/unfollow", a.isAuthorized(a.UnfollowEndpoint()))
+
 	router.POST("/timeline", a.isAuthorized(a.TimelineEndpoint()))
-	router.POST("/upload", a.isAuthorized(a.UploadMediaEndpoint()))
+	router.POST("/discover", a.DiscoverEndpoint())
+
 	router.GET("/profile/:nick", a.ProfileEndpoint())
+	router.POST("/profile/:nick/twts", a.ProfileTwtsEndpoint())
```
Comment (dooven) [inline internal/api.go:80]: Im starting to think i should've named this `/user/*` :thinking:

Comment (prologic) [reply internal/api.go:80]: Probably :) It matches the Web handlers then!

File: internal/api.go [hunk 2]
```diff
@@ -863,6 +869,76 @@ func (a *API) ProfileEndpoint() httprouter.Handle {
 	}
 }
 
+// ProfileTwtsEndpoint ...
+func (a *API) ProfileTwtsEndpoint() httprouter.Handle {
```
Comment (prologic) [inline internal/api.go:873]: Maybe its better to call this the `FeedEndpoint` that is capable of accepting either a local user or external feed by slug?

**Review by prologic — APPROVED** [review]
Plenty of time to refactor things!

**Pull request merged**